sound/soc/sof/ipc4-loader.c

```diff
@@ -482,13 +482,10 @@ void sof_ipc4_update_cpc_from_manifest(struct snd_sof_dev *sdev,
 		msg = "No CPC match in the firmware file's manifest";
 
 no_cpc:
-	dev_warn(sdev->dev, "%s (UUID: %pUL): %s (ibs/obs: %u/%u)\n",
-		 fw_module->man4_module_entry.name,
-		 &fw_module->man4_module_entry.uuid, msg, basecfg->ibs,
-		 basecfg->obs);
-	dev_warn_once(sdev->dev, "Please try to update the firmware.\n");
-	dev_warn_once(sdev->dev, "If the issue persists, file a bug at\n");
-	dev_warn_once(sdev->dev, "https://github.com/thesofproject/sof/issues/\n");
+	dev_dbg(sdev->dev, "%s (UUID: %pUL): %s (ibs/obs: %u/%u)\n",
+		fw_module->man4_module_entry.name,
+		&fw_module->man4_module_entry.uuid, msg, basecfg->ibs,
+		basecfg->obs);
```

Comment:
The reason why it ended up as dev_warn() + dev_warn_once() is to make sure it _shows_ up in the logs and 'annoy' users that they will file a bug so the CPC value is fixed. Initially I had dev_dbg() as well.
Do we need to print it every time a stream starts? Would a dev_dbg_once() can suffice? Well, it is dev_dbg(), so only developers going to see it, right?

Reply:
I think the original intent still has merit. It's just we need to scale back on the "annoy dose" given the fixes come via quarterly SOF releases (plus adoption to upstream). You can add or modify a tplg at any time, but you can't get rid of the warnings without a new (typically signed) image.